### PR TITLE
Add findSTRINGKnode function

### PR DESCRIPTION
**Pull request by rn1n3r**
Add exploratory system analysis functionality to investigate the strength of association of vertices in the STRING graph provided by Dr. Steipe to the curated genes provided in the system analysis. Based on work by Cornish & Markowetz (2014) https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003808.

### CODE REVIEW

File: R/findSTRINGKnodes.R [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#' a measure of the vertex association to the high-weighted vertices defined by the input
+#' curated gene system 'sys'. Also plots the knode values for the top percentile of knode values
+#' given by 'per'
+#'
+#' @param sys (character)       5-character system code of genes
+#' @param per (numeric)         Percentile of results to be plotted. Default is 0.05
+
+#' @return (numeric) Knode values
+#' @examples
+#' findSTRINGKnodes("PHALY", 0.05)
```
Comment (sapirl) [inline R/findSTRINGKnodes.R]: A short explanation for the example is missing

**Review by hyginn — CHANGES_REQUESTED** [review]
I'm just adding a request-changes tag here, to flag the request as not-quite-ready-yet.

File: vignettes/findSTRINGKnodes.bib [hunk 2]
```diff
@@ -0,0 +1,42 @@
+@misc{steipe-rptPlus,
```
Comment (hyginn) [inline vignettes/findSTRINGKnodes.bib]: Merge into one single package bib. Removing this from the fork.

**Review by hyginn — APPROVED** [review]
Some details to be changed after merge

**Comment by hyginn** [comment]
Hm. Can't resolve deleted Description - merging manually. Closed.

**Pull request closed without merging**